packages/esm-patient-immunizations-app/src/immunizations/immunizations-form.component.tsx

```diff
@@ -40,17 +51,44 @@ const ImmunizationsForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, close
     formChanged: false,
   };
   const [formState, setFormState] = useState(initialState);
-  const updateSingle = <T extends keyof ImmunizationFormState>(name: T, value: typeof formState[T]) =>
+  const updateSingle = useCallback(<T extends keyof ImmunizationFormState>(name: T, value: typeof formState[T]) => {
```

Comment:
We don't require to give the type for `useCallbacks`. We can directly give the type to the expected arguments.